### ISSUE TITLE
Can you support `merge_group` event in plan-preview-action ?

### ISSUE THREAD

**Issue by TakumaKurosawa** [pipe-cd/pipecd#4854]
**What would you like to be added**:

I would like you to support the `merge_group` event in plan-preview-action.

**Why is this needed**:

My project is using [merge queue](https://docs.github.com/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue).

Merge queue is failed with this error:
> unexpected event merge_group, only "pull_request", "issue_comment" and "push" event are supported

**Reference**:

https://github.com/pipe-cd/pipecd/blob/7b471d7a9f2eb09ccbc0a9222b6e7adb08ea2edf/tool/actions-plan-preview/main.go#L231-L233

**Comment by niladrix719** [comment]
Hey, will try to raise a Pr for this

**Comment by t-kikuc** [comment]
@niladrix719 
thank you for your interest, but please let us assign another developer.
That's because good FIRST issues are intended for educating a new contributor. Please keep opportunities for other developers.

When new contributors got interested in this issue, please help them 👍

**Comment by t-kikuc** [comment]
@niladrix719 
Of course, just investigating and providing hints of development here is welcome!
Please let someone create a PR.

**Comment by Tony-Stark20** [comment]
I would like to work on this

**Comment by eeshaanSA** [comment]
Thanks Tushar, you can take it!
